OurUmbraco.Site/Views/Partials/Projects/ListProjects.cshtml

```diff
@@ -158,6 +165,7 @@
     }
 }
 <h1>Browse @headline packages</h1>
+<p><em>A total of @total packages match your current criteria</em></p>

```

Comment:
If there was only 1 package that matched the search criteria, this would display as "1 packages".  Would be nice if it displayed as "1 package" in that scenario.